src/lib/containers/GenericCard.tsx

```diff
@@ -53,17 +58,33 @@ export const DOI_REGEX = /^10.\d{4,9}\/[-._;()/:a-z0-9]+$/
 // check for 'syn' followed and ended by a digit of unlimited length, must also begin the line
 export const SYNAPSE_REGX = /^syn\d+$/
 
+// This function isn't in the class only for ease of testing with renderShortDescription
+export const getCutoff = (summary: string) => {
+  let previewText = ''
+  const summarySplit = summary!.split(' ')
+  // find num words to join such that its >= char_count_cutoff
+  let i = 0
+  while (previewText.length < CHAR_COUNT_CUTOFF && i < summarySplit.length) {
+    previewText += `${summarySplit[i]} `
+    i += 1
+  }
+  previewText = previewText.trim()
+  return { previewText }
```

Comment:
Is there a reason why you're returning an object with previewText instead of just previewText?  Certainly would make sense if there were multiple return values...